tests/acceptance/repo/settings-test.js

```diff
@@ -35,6 +36,14 @@ module('Acceptance | repo settings', function (hooks) {
         admin: true
       }
     });
+
+    const mockPermissionsService = Service.extend({
+      hasPushPermission() {
+        return true;
+      }
+    });
+    stubService('permissions', mockPermissionsService);
```

Comment:
I think it's a bad idea to stub the service here. The purpose of acceptance tests is to verify that the app works properly in real environment. If tests are failing, then something is not working properly. Stubbing the service only hides the error by excluding the problematic area.

Reply:
Many many thanks for your support @artursmirnov !!!